rust/agents/relayer/src/msg/processor.rs

```diff
@@ -21,21 +19,21 @@ use super::SubmitMessageArgs;
 #[derive(Debug)]
 pub(crate) struct MessageProcessor {
     db: AbacusDB,
-    inbox_contracts: InboxContracts,
+    mailbox: CachingMailbox,
```

Comment:
may be nice to clarify somewhere that this is the destination mailbox (maybe this is immediately clear to readers though ?)